### PR TITLE
enh(drawio): allow user to edit pie chart properties

### PR DESCRIPTION
**Pull request by Yassir-BenBOUBKER**
add the possibility to change Pie chart properties.
image

### CODE REVIEW

**Review by Thebarda — APPROVED** [review]

File: src/main/webapp/js/grapheditor/Format.js [hunk 1]
```diff
@@ -4385,6 +4387,49 @@ StyleFormatPanel.prototype.init = function()
...
+		var cells = ss.cells;
+		const numberOfPieChartSelected = cells.filter((cell) => cell.getAttribute('type') === 'WIDGET' && cell.getAttribute('widgetType') === 'PIE_CHART').length;
+
+		if(numberOfPieChartSelected === cells.length)
+		{
+			
+			const viewerStyles =  this.createPanel('')
+			viewerStyles.style.paddingTop = '6px';
+			viewerStyles.style.paddingBottom = '8px';
+			// viewerStyles.appendChild(this.createTitle(mxResources.get('Viewer Styles')));
```
Comment (bdauria) [inline src/main/webapp/js/grapheditor/Format.js]:
```suggestion
```

**Comment by Yassir-BenBOUBKER** [comment]
@bdauria ou @Thebarda do i merge this PR or it better to do it in React

**Comment by Thebarda** [comment]
> @bdauria ou @Thebarda do i merge this PR or it better to do it in React

For maintenance reasons, It would be better in React. Is it difficult to integrate it in the React app?

**Comment by Yassir-BenBOUBKER** [comment]
i am just asking so i can close this PR

> > @bdauria ou @Thebarda do i merge this PR or it better to do it in React
> 
> For maintenance reasons, It would be better in React. Is it difficult to integrate it in the React app?

No, not at all
because i thinks we will merge to monorepo next week
i stop creating new PRs, and i am closing others
to create them later in monorepo

**Comment by Thebarda** [comment]
> No, not at all because i thinks we will merge to monorepo next week i stop creating new PRs, and i am closing others to create them later in monorepo

Okay, very nice. Wise decision 👍

**Comment by Yassir-BenBOUBKER** [comment]
closed: must be done on react and waiting for monorepo

**Pull request closed without merging**